kits/automation/time-series-preprocessor/README.md

```
@@ -0,0 +1,162 @@
+└── README.md
+```
+
+---
+
+## Contributing
+
+Contributions are welcome. Open an issue or pull request in the [AgentKit repository](https://github.com/Lamatic/AgentKit).
+
+## License
+
+MIT License — see [LICENSE](../../../../LICENSE).
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)